x-pack/plugins/fleet/server/routes/app/index.ts

```diff
@@ -7,9 +7,12 @@
 
 import type { IRouter, RequestHandler } from 'src/core/server';
 
-import { APP_API_ROUTES } from '../../constants';
+import { PLUGIN_ID, APP_API_ROUTES } from '../../constants';
 import { appContextService } from '../../services';
-import type { CheckPermissionsResponse } from '../../../common';
+import type { CheckPermissionsResponse, RegenerateServiceTokenResponse } from '../../../common';
+import { defaultIngestErrorHandler, RegenerateServiceTokenError } from '../../errors';
+
+const DEFAULT_FLEET_SERVER_SERVICE_TOKEN_NAME = 'default-fleet-server-token';
```

Comment:
The service account is already called fleet-server and it is tokens for it, so not sure we need to include it all in the name. Thinking of a future where there are multiple tokens, how will the user differentiate between these. Should we call it `token-{number}` for `token-{timestamp}` for now and later let the user specify a name if they want?